src/lib/Commands/TestCommand/index.js

```diff
@@ -49,6 +72,16 @@ const DEFAULT_EXTRA_TEST_MESSAGE_TIMEOUT = 5;
 const BUILD_API_KEY_ENV_VAR = 'IMP_BUILD_API_KEY';
 
 /**
+ * Unsupported in Builder symbols
+ */
+const UNSUPPORTED_SYMBOLS_REGEXP = /[^_A-Za-z0-9]/g;
```

Comment:
Are we aware of any other characters used in the existing tests? I would really doubt it. Even so, it's better to fix the tests rather than to invent a generic solution for this case here.